src/land/UberArcanistSubmitQueueEngine.php

```diff
@@ -25,15 +26,20 @@ public function execute() {
       $this->updateRevision();
 
       $this->pushChange();
-      $this->reconcileLocalState();
-
-      if ($this->getShouldKeep()) {
-        echo tsprintf(
-        "%s\n",
-        pht('Keeping local branch.'));
+      if ($this->shouldShadow) {
```

Comment:
hmm this is actually our class? Since this is something only we control, I would like to optimize on readability.